docs/content/reference/api.md

```diff
@@ -906,7 +906,7 @@ string
 </em>
 </td>
 <td>
-<p>InstanceProfile is TODO</p>
```

Comment:
@gregsheremeta this is generated from the godoc on the type in api/v1alpha1/nodepool_types.go, you need to change it there and then run `make api` and `make api-docs`

Reply:
Yeah, this is the one file that's generated. I put a header [at the top](https://raw.githubusercontent.com/openshift/hypershift/main/docs/content/reference/api.md):

```
---
################# IMPORTANT ########################
# Document generated by `make api-docs`. DO NOT EDIT
################# IMPORTANT ########################
```

but maybe I should have made it bigger 😁 

https://hypershift-docs.netlify.app/contribute/contribute-docs/#generate-the-api-reference

Reply:
Now that I've mentioned it, I should be able to detect edits in the `make verify` step...

Reply:
https://github.com/openshift/hypershift/pull/853